### PR TITLE
Add `Skinning Contest: Icons of an Era` news post

### PR DESCRIPTION
**Pull request by 0x84f**
yeah woohoo awesome

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] re-time or re-date if necessary, if re-dating make sure the contest start is also changed in the news post

### CODE REVIEW

File: news/2023/2023-05-10-skinning-contest-icons-of-an-era.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+layout: post
+title: "Skinning Contest: Icons of an Era"
+date: 2023-05-10 12:00:00 +0000
+---
+
+Celebrate 15 years of osu! by expressing your creativity through osu! skinning!
+
+![](https://assets.ppy.sh/contests/178/header.jpg)
+
+A lot has happened in the 15 years since osu!'s inception — and every time the community thought that we have reached the top, whether that'd be in mapping, skinning or gameplay, the boundaries of what is deemed possible kept getting pushed back further and further. So without further ado, let's go over the theme.
```
Comment (Walavouchey) [inline news/2023/2023-05-10-skinning-contest-icons-of-an-era.md:11]: a lot of words to say "boundaries have been pushed", but to me it doesn't seem a whole lot relevant to the contest nor skinning specifically.. ?

Comment (0x84f) [reply news/2023/2023-05-10-skinning-contest-icons-of-an-era.md:11]: It's something I wrote at the very inception of the news post so can probably revise it, it really only serves a purpose to fluff up the post and I thought it read nice.

I'll look to change it to something more relevant

Comment (0x84f) [reply news/2023/2023-05-10-skinning-contest-icons-of-an-era.md:11]: addressed in https://github.com/ppy/osu-wiki/pull/9335/commits/afeb1acb6cbe79e43ca246f5679ad582ff98f006 let me know if this is better

File: news/2023/2023-05-08-skinning-contest-icons-of-an-era.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+After the submission phase is over in roughly 35 days time (from the date of posting this article), the entries will be made available for community voting. Make sure to get involved when the time comes!
+
+The voting will stay open for 14 days. Once that passes, the top 3 entries will become eligible for prizes as described below.
+
+## Schedule
+
+| Event | Time |
+| :-- | :-- |
+| Submission phase | 2023-05-10 |
```
Comment (RockRoller01) [inline news/2023/2023-05-08-skinning-contest-icons-of-an-era.md]:
```suggestion
| Submission phase | 2023-05-10/2023-07-14 |
```
Its not just a single day, but rather 5 weeks from here, so until 14.7.2023

File: news/2023/2023-05-10-skinning-contest-icons-of-an-era.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+The voting will stay open for 14 days. Once that passes, the top 3 entries will become eligible for prizes as described below.
+
+## Schedule
+
+| Event | Time |
+| :-- | :-- |
+| Submission phase | 2023-05-10 |
+| Voting phase | 2 weeks starting from a separate news post |
+| Results | 2023-06-14 |
```
Comment (RockRoller01) [inline news/2023/2023-05-10-skinning-contest-icons-of-an-era.md:43]: There is also no fixed date for results currently, this should just be TBD

File: news/2023/2023-05-10-skinning-contest-icons-of-an-era.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+![](https://assets.ppy.sh/contests/178/header.jpg)
+
+Over the course of osu!'s history there have been many memories made, whether that'd be related to maps that would become legendary within the community, plays that to this day remain unbeaten and are considered the best of all time, or the various community-made creations that have been popularised throughout the years.
+
+We wanted this contest's theme to not only centre around some of those memories, but also provide plenty of material that could inspire the skinning community to take part and provide us with some amazing skins!
+
+## Theme: Icons of an Era
+
+This year's theme, **Icons of an Era**, aims to spark the community's imagination by creating art around various iconic osu! beatmaps that were crafted and risen to glory throughout osu!'s lifetime, as well as the many either retired, official or community-made osu! mascots that have been introduced over the course of the past 15 years.
```
Comment (RockRoller01) [inline news/2023/2023-05-10-skinning-contest-icons-of-an-era.md:17]: Probably shouldn't write "this years theme" as we plan to have a second contest this year

File: news/2023/2023-05-10-skinning-contest-icons-of-an-era.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+Whether your entry is focused around pippi singing into the microphone from [The Big Black](https://osu.ppy.sh/beatmapsets/41823#osu/131891), a group of beatmap backgrounds from a specific era in osu! history, or just a skin focused around objectively the best mascot that has ever grazed osu! (Taikonator 4000), you will need to make sure that the theme is prominent as anyone should be able to open your entry and almost immediately recognise what the inspiration behind your entry was.
```
Comment (Walavouchey) [inline news/2023/2023-05-10-skinning-contest-icons-of-an-era.md:19]: https://osu.ppy.sh/wiki/en/Mascots#taikonator

```suggestion
Whether your entry is focused around pippi singing into the microphone from [The Big Black](https://osu.ppy.sh/beatmapsets/41823#osu/131891), a group of beatmap backgrounds from a specific era in osu! history, or just a skin focused around objectively the best mascot that has ever grazed osu! (Taikonator 3000), you will need to make sure that the theme is prominent as anyone should be able to open your entry and almost immediately recognise what the inspiration behind your entry was.
```

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**